.github/pr-labeler.config.yml

```diff
@@ -1,3 +1,12 @@
-# Additional labels to assign based on file changes
-"documentation":
-  - documentation/**/*
+# Patterns to match to auto-assign labels based on path changes
+# https://github.com/actions/labeler/tree/v5/?tab=readme-ov-file#match-object
```

Comment:
This is nicely extendable. I can see how a `frontend` label could also be useful (or even a `component` label for files in `src/app/shared/components/template/components/**/*`), but as you say in the PR description, such a label would likely not be applied in all cases and so may therefore have limited usefulness